src/hb-ot-layout.cc

```diff
@@ -1047,6 +1055,48 @@ hb_ot_layout_get_size_params (hb_face_t    *face,
 }
 
 
+/**
+ * hb_ot_layout_get_stylistic_set_label:
+ *
+ * Since: REPLACEME
+ **/
+int
+hb_ot_layout_get_stylistic_set_label (hb_face_t    *face,
```

Comment:
It should be `_get_feature_label_name_id` or something since it returning the name IDs of the label, not the actual label itself.